GoogleForCreators/web-stories-wp
# Frontend: Add default social sharing configuration

## Summary

Adds a default social sharing configuration with a reasonable list of providers.

The list can be adapted and sorted using a new filter.

## Relevant Technical Choices

Figured it makes sense for this to live next to `print_analytics`

## To-do



## User-facing changes

<img width="434" alt="Screenshot 2020-12-22 at 18 00 27" src="https://user-images.githubusercontent.com/841956/102914322-640cdf80-4480-11eb-978c-309b68f007fe.png">

## Testing Instructions

1. Create new story
2. View story on frontend
3. Click on share button and see new list of sharing providers

---



Fixes #5780
